src/registrar/assets/js/get-gov.js

```diff
@@ -834,3 +834,436 @@ function hideDeletedForms() {
+                <use xlink:href="/public/img/sprite.svg#delete"></use>
+              </svg> Delete <span class="usa-sr-only">${domainName}</span>
+            </a>` : '';
+
+          const row = document.createElement('tr');
+          row.innerHTML = `
+            <th scope="row" role="rowheader" data-label="Domain name">
+              ${domainName}
+            </th>
+            <td data-sort-value="${new Date(request.submission_date).getTime()}" data-label="Date submitted">
+              ${submissionDate}
+            </td>
+            <td data-label="Status">
+              ${request.status}
+            </td>
+            <td>
+              <a href="${actionUrl}">
+                <svg class="usa-icon" aria-hidden="true" focusable="false" role="img" width="24">
+                  <use xlink:href="/public/img/sprite.svg#${request.state === 'deleted' || request.state === 'on hold' ? 'visibility' : 'settings'}"></use>
+                </svg>
+                ${actionLabel} <span class="usa-sr-only">${request.requested_domain ? request.requested_domain : 'New domain request'}</span>
+              </a>
+            </td>
+            <td>${deleteButton}</td>
+          `;
+          tbody.appendChild(row);
+        });
+        // initialize modals immediately after the DOM content is updated
+        initializeModals();
+        if (loaded)
+          ScrollToElement('id', 'domain-requests-header');
+
+        hasLoaded = true;
+
+        // update the pagination after the domain requests list is updated
+        updateDomainRequestsPagination(data.page, data.num_pages, data.has_previous, data.has_next, data.total);
+        currentSortBy = sortBy;
+        currentOrder = order;
+      })
+      .catch(error => console.error('Error fetching domain requests:', error));
+  }
+
+  /**
+   * Update the pagination below the domain requests list.
+   * @param {*} currentPage - the current page number (starting with 1)
+   * @param {*} numPages - the number of pages indicated by the domain request list response
+   * @param {*} hasPrevious - if there is a page of results prior to the current page
+   * @param {*} hasNext - if there is a page of results after the current page
+   */
+  function updateDomainRequestsPagination(currentPage, numPages, hasPrevious, hasNext, totalItems) {
+    // identify the DOM element where pagination is contained
+    const counterContainer = document.querySelector('#domain-requests-pagination .usa-pagination__counter');
+    const paginationContainer = document.querySelector('#domain-requests-pagination .usa-pagination__list');
+    counterContainer.innerHTML = '';
+    paginationContainer.innerHTML = '';
+
+    // pagination should only be displayed if there are more than one pages of results
+    paginationContainer.classList.toggle('display-none', numPages <= 1);
+
+    counterContainer.innerHTML = `${totalItems} Domain request${totalItems > 1 ? 's' : ''}`;
```

Comment:
**non blocking comment** should this be focusable? wondering if that would make navigating to it and having it read via voice over easier.

Reply:
Interesting question. How about we move this counter into the `<h2>`?

Reply:
Deferring to design comment, below